plugins/inputs/cloud_pubsub/pubsub.go

```diff
@@ -44,7 +44,8 @@ type PubSub struct {
 
 	Base64Data bool `toml:"base64_data"`
 
-	Log telegraf.Logger
+	Log             telegraf.Logger
+	ContentEncoding string `toml:"content_encoding"`
```

Comment:
Only cosmetics, but if you don't mind to swap the two lines and add a toml tag to `Log` such as
```suggestion
	ContentEncoding string `toml:"content_encoding"`
	Log             telegraf.Logger `toml:"-"`
```
to avoid `Log` being set by the toml parser by accident.